### ISSUE TITLE
Clicking Shift symbol key should also change case of swyped word

### ISSUE THREAD

**Issue by lamyergeier** [futo-org/android-keyboard#129]
In the following image when I press up arrow key in the keyboard the suggestion should change from **paid** to **Paid** (this is a feature in Swift keyboard)

![Screenshot_20240702_201133_Fennec](https://github.com/futo-org/android-keyboard/assets/42092626/f171ce62-777a-4ce0-833b-f6dc27ef9c32)

**Comment by NagBoard** [comment]
Make this an **option**, please

**Comment by abb128** [comment]
Currently you can already change the case by highlighting the word first (double tap it in the text field)

**Comment by lamyergeier** [comment]
May be space bar functionality can be added as in Swiftkey as it is more easy and offers additional functionality as described in original post

**Comment by dreamingspider28** [comment]
yes please I love this keyboard but this is one thing that would make it a keeper for me 🙏
